test/entity/epp-status-code-test.php

```diff
@@ -51,17 +51,16 @@ class EPP_Status_Code_Test extends Test\Lib\Domain_Services_Client_Test_Case {
 	];
 
 	public function test_entity_instance_success_disclose_all(): void {
-		foreach ( self::VALID_EPP_STATUSES as $status => $is_updateable ) {
+		foreach ( self::ICANN_EPP_STATUSES as $status => $is_updateable ) {
 			$entity = new Entity\Epp_Status_Code( $status );
 			$this->assertSame( $status, (string) $entity );
 			$this->assertSame( $is_updateable, $entity->is_updateable() );
+			$this->assertTrue( $entity->is_icann_status() );
 		}
 	}
 
 	public function test_entity_instance_fail(): void {
```

Comment:
Maybe we could rename this test to `test_entity_instance_not_icann_status` or something similar?